api/src/main/kotlin/com/google/devtools/ksp/utils.kt

```diff
@@ -241,6 +242,21 @@ fun KSDeclaration.isVisibleFrom(other: KSDeclaration): Boolean {
     }
 }
 
+@Throws
+fun KSFunctionDeclaration.getThrowList(): Sequence<KSType> {
```

Comment:
can we make this function handle both cases so that developer does not need to check the origin before calling?

Reply:
I can do that, but such function will have to be either a member function of `Resolver` or a utility function that takes a resolver as parameter, as resolver is needed to check throw list on Java origin.

Reply:
+1 on unifying them. Otherwise it would be weird to get an empty sequence when calling this on a Java function.

Putting it in `Resolver` sounds better to me, since this is more likely to be used when processing Java or Kotlin sources that need to be Java compatible. It's easier to be organized together with other Java specific functions such as `getJvmName`.